routes.js

```diff
@@ -39,6 +39,9 @@ module.exports = function (app, router) {
     //route for getting pokemon details by specific name
     router.get('/pokemon/name/:name', pokemon.getByName);
 
+    //route for getting pokemon details by description
+    router.get('/pokemon/description/:description', pokemon.getByDescription());
```

Comment:
I think this needs to be without "()" and implemented in
https://github.com/PokemonGoers/PokeData/blob/8208525bbac1c592e1d6d1a651ee8e51daf19c05/app/stores/pokemon.js ?

Can this be implemented like the "getByName" so we can e.g. search for "Seed" and get "Bulbasaur" as result with "can be seen napping in bright sunlight. There is a seed..."? :)